_posts/2025-01-16-wamr-2024-summary.md

```diff
@@ -0,0 +1,82 @@
+- WASI-NN: Support for WASI-NN with OpenVINO and llama.cpp backends has been introduced.
+- WASI Support: Ongoing support for WASI on ESP-IDF and Zephyr.
+- Table64: Table64 support for the interpreter and AOT has been finalized.
+
+These new features and improvements are designed to make WAMR more powerful and easier to use, catering to the needs of developers and industry professionals alike.
+
+## Active engagement in Embedded Special Interest Group (ESIG)
+
+In the embedding industry, the perspective on Wasm differs slightly from the cloud-centric view that the current Wasm Community Group (CG) often focuses on. To address these unique requirements, the Embedded Special Interest Group (ESIG) was established in 2024. This group aims to discover solutions that prioritize performance, footprint and stability, tailored specifically for embedding devices.
+
+The ESIG has already achieved several accomplishments this year, thanks to the shared understanding and collaboration with customers. By focusing on the unique needs of the embedding industry, ESIG is paving the way for more specialized and efficient Wasm solutions.
+
+## Industrial adoption
+
+The adoption of WAMR in the industry has been remarkable, with several key players integrating WAMR into their systems to leverage its performance and flexibility. Here are some notable examples:
+
+Alibaba's Microservice Engine (MSE) has adopted WAMR as a Wasm runtime to execute Wasm plugins in their gateways Higress. This integration has resulted in an [impressive ~50% performance improvement](https://www.alibabacloud.com/blog/higresss-new-wasm-runtime-greatly-improves-performance_601025), showcasing the efficiency and robustness of WAMR in real-world applications.
+
+WAMR has also been integrated into Runwasi as one of the Wasm runtimes to execute Wasm in containerd. This integration allows for seamless execution of Wasm modules within containerized environments, providing a versatile and efficient solution for running Wasm applications.
+
+For more information on industrial adoptions and other use cases, please refer to [this link](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/ADOPTERS.md).
+
+These examples highlight the growing trust and reliance on WAMR in various industrial applications, demonstrating its capability to deliver significant performance enhancements and operational efficiencies.
+
+## Conclusion
+
+2024 has been a transformative year for WAMR, marked by significant community contributions, innovative features, and the establishment of the ESIG. As we look ahead, we are excited about the continued growth and evolution of WAMR, driven by the passion and dedication of our community. We invite you to join us on this journey, explore the new features, and contribute to the future of WebAssembly Micro Runtime.
+
+Thank you for being a part of the WAMR community. Here's to an even more exciting 2025!

```

Comment:
It'd be good to link here the 2025 roadmap (if there's one already)

Reply:
Apologies, but we don't have such a roadmap at the moment. In fact, we are thinking about creating it collaboratively with the community. 

In 2025, the Intel team may need to allocate most of their resources to an internal project, and their contributions to WAMR might focus more on quality aspects like fuzzing, rather than on new features. They will also continue to seek a solution for shared memory with the host.